elastic/kibana
# [Uptime] [7.4.1] Add fix for empty name filter

## Summary

Resolves https://github.com/elastic/kibana/issues/46744.

Adds name filtering to the list of supported values.

## Testing
0. Start up this patch with Elasticsearch and Heartbeat. Make sure your HB config has several monitors with names and several without names.
1. Navigate to the Uptime UI. Ensure that when you open the names dropdown list, you see the list of names you expect. You should not see any entries besides the names you've defined.
